<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HtmlIdGenDlg</class>
 <widget class="QDialog" name="HtmlIdGenDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>生成随即字串</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="randomSerialsBox">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>配置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelPrex">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>前缀字串：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="prexLineEdit">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>可选</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="labelPost">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>后缀字串：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="postLineEdit">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>xhtml</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>位数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lenLineEdit">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>字词：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ctxLineEdit">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>可选</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>ctxLineEdit</zorder>
     <zorder>label_2</zorder>
     <zorder>postLineEdit</zorder>
     <zorder>labelPost</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>208</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnOK">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancle">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
